openai-assistant/target-platform/ivy.target

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="ivy">
+	<locations>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://p2.axonivy.com/p2/11.3/"/>
```

Comment:
why the 11.3 platform? the code currently is written for compatibility with LTS 10

Reply:
I'm not sure why, but when I'm using LTS 10, I get this error:
![image](https://github.com/user-attachments/assets/d2eaa069-68da-443f-8716-a8b0e1b662ea)

Reply:
Yes, the APIs may are no longer the same as they are internal.
However, by using a newer platform you may get this to compile, but it will fail anyway at runtime on a Desinger10.

So either you find an API that works on 10 and 11.3 ... or you actually migrate to 11.3 and mark the artifacts as such ... with the downside that you can no longer ship anything for lts10.